data/brands/shop/convenience.json

```diff
@@ -3692,6 +3702,16 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Woolworths Metro",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "brand": "Woolworths Metro",
+        "brand:wikidata": "Q111772555",
+        "name": "Woolworths Metro",
+        "shop": "convenience"
```

Comment:
In my opinion these are fully fledged shop=supermarket, the wiki describes convenience as "A small local shop carrying a small subset of the items you would find in a supermarket", at least the Woolies Metro I go to has pretty much everything compared to the full Wollies, just lacking the deli.